src/main/webapp/index.html

```diff
@@ -4,21 +4,24 @@
     <meta charset="UTF-8">
     <title>Air Quality Map</title>
     <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
   </head>
   <body>
     <!-- Sidebar -->
-    <div class="sidenav">
+    <div class="sidenav" id="sidebar">
```

Comment:
since we're going with calling this "feature" side bar, should we change the naming of the class to sidebar too for consistency?

Reply:
Yep, or the alternative would be to just use the sidebar id in the css rather than a class since we only have one sidebar element

Reply:
sure, for it to be changed  to sidebar id as opposed to class - gotta edit the css

Reply:
Done